components/Auth/SignIn.js

```diff
@@ -65,17 +65,25 @@ class SignIn extends Component {
       email: props.email || '',
       polling: false,
       loading: false,
-      success: undefined
+      success: undefined,
+      cookiesEnabled: true
+    }
+  }
+
+  componentDidMount () {
+    if (navigator) {
```

Comment:
This check is a bit pointless, running it in node would explode:
```
> if (navigator) console.log(123)
ReferenceError: navigator is not defined
```

But `componentDidMount` is always only run in the browser. No need for checking weather a navigator object exists here.

Reply:
Will do.